selftests/functional/test_basic.py

```diff
@@ -1034,12 +1043,9 @@ def run_config(config_path):
 
         result_plugins = ["json", "xunit", "zip_archive"]
         result_outputs = ["results.json", "results.xml"]
-        try:
-            pkg_resources.require('avocado_result_html')
```

Comment:
This change of how we check things is not needed.  The failure is caused by another issue: the names are wrong.  For the HTML plugin, for instance, it should be `avocado-framework-plugin-result-html`, as per the Python package name (and not as per the Python module name).

Reply:
I thought it's something like that, but I checked it with `avocado_framework_plugin...`. I'll change it in v2.